tests/suite/test_virtual_server_grpc.py

```diff
@@ -186,6 +186,21 @@ def test_config_after_enable_tls(self, kube_apis, ingress_controller_prerequisit
                                             ingress_controller_prerequisites.namespace)
         assert 'grpc_pass grpcs://' in config
 
+    @pytest.mark.ciara
+    @pytest.mark.parametrize("backend_setup", [{"app_type": "grpc-vs"}], indirect=True)
+    def test_config_error_page_warning(self, kube_apis, ingress_controller_prerequisites, crd_ingress_controller, 
+                                       backend_setup, virtual_server_setup):
+        text = f"{virtual_server_setup.namespace}/{virtual_server_setup.vs_name}"
+        vs_event_warning_text = f"Configuration for {text} was added or updated ; with warning(s): The error page"
+        patch_virtual_server_from_yaml(kube_apis.custom_objects,
+                                        virtual_server_setup.vs_name,
+                                        f"{TEST_DATA}/virtual-server-grpc/virtual-server-error-page.yaml",
+                                        virtual_server_setup.namespace)
+        wait_before_test(30)
```

Comment:
is 30s wait really necessary?

Reply:
Good catch! I had 3 seconds which was flaky and I added a 0 just to check - I'll change to 5.